### ISSUE TITLE
Limesurvey integration

### ISSUE THREAD

**Issue by gerhardriener** [beliefelicitation/qualtrics#1]
I was trying to integrate the javascript in LimeSurvey, as this is the only survey software I have access to, and I tried to reverse engineer the HTML code for the question used in Qualtrics. As the free version of Qualtrics does not allow the use of javascript, I could not have a closer look at what the HTML code of the questions looks like. Could you please upload a complete sample of the HTML question code used in Qualtrics, then, I will try to develop a LimeSurvey version.

**Comment by ismaelbenslimane** [comment]
Here is a test set up with limesurvey, here is a limesurvey template you can import, does it work for you? [https://github.com/beliefelicitation/qualtrics/blob/main/limesurvey_survey_192993.lss](https://github.com/beliefelicitation/qualtrics/blob/main/limesurvey_survey_192993.lss)

**Comment by gerhardriener** [comment]
Great, thank you. It worked, I tweaked it a bit, making a question and not a question group and survey and question ids relative, so that it is easy to include it to any survey. Moreover, I used the question type "Multiple short text" and I set the answer fields as hidden via javascript. I only do not know how long the DistributionHistory field can get and whether it fits in a short text field.
[limesurvey_question_287800.lsq.zip](https://github.com/beliefelicitation/qualtrics/files/10050737/limesurvey_question_287800.lsq.zip)
